package.json

```diff
@@ -29,11 +29,10 @@
     "canjs-plugin",
     "donejs"
   ],
-  "system": {
```

Comment:
When that configuration is something that is needed by the app using the library. For example, in steal-stache it configures itself to be the plugin for the stache `ext` config. So I would use `system` for backwards compat.

However, when the configuration is only used for local development of the library, as in this case, then updating to `steal` is fine.